.github/workflows/integration-test.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         with:
             repository: layr-labs/eigensdk-go
             token: ${{ github.token }}
-            ref: dev
+            ref: 37139a7d8ea870ca2c36e6aca72115de12c26d5d
```

Comment:
Can you share context on this?

Reply:
this is the eigensdk commit from tmp-slashing branch since that has updated anvil state.